packages/frontend-shared/src/gql-components/topnav/TopNav.vue

```diff
@@ -97,7 +97,7 @@
       class="h-16px w-16px group-hocus:icon-dark-indigo-500 group-hocus:icon-light-indigo-50 icon-dark-gray-500 icon-light-gray-100"
     />
     <span>
-      v{{ versions.latest.version }}
+      v{{ versions.current.version }}
```

Comment:
Switching to `current` does feel slightly safer, even though this is only shown if we are not running an old version - so `current` and `latest` should be the same.